script/generate_licenses.py

```diff
@@ -233,12 +233,12 @@ def main():
         'background images used on the new tab page. The Brave Browser and ' \
         'such data files are separate and independent works.'
 
-    ntp_backgrounds = list_ntp_backgrounds(
-        os.path.join(ntp_data_dir, 'backgrounds.ts'))
-    if write_license_file(ntp_data_dir, generate_backgrounds_license(
-            ntp_backgrounds_preamble, ntp_backgrounds)):
-        print(f'- {len(ntp_backgrounds)} sub-components added in ' \
-              'brave_new_tab_ui/data/LICENSE')
+    # ntp_backgrounds = list_ntp_backgrounds(
```

Comment:
try to remove only the things not needed. if this is already removed from react then need not comment this